CONTRIBUTING.md

```diff
@@ -90,6 +90,14 @@ and add the label_pipelines_appstudio_openshift_io_type to it.
 Signed-off-by: Avi Biton <abiton@redhat.com>
 ```
 
+### Pull Request Description
+The PR description (the first comment at the top of the PR) can usually be derived from
+the commit message body - especially if the PR consists of a single commit.
+
+On top of that, the PR description should include a bit of context, so that reviewers
```

Comment:
I think this is more relevant to the commit message as PR description should just mirror it IMO

Reply:
As I see it, the commit messages are a record of the changes in the repo. I agree that the content is very helpful for PR descriptions as well, but during the sprint review, some mentioned that they are still missing some content on the descriptions to allow effective review.

I'm trying here to convey that further explanation about the context should go inside the PR description.